2021-12-09 18:32:10 - INFO - "GET / HTTP/1.1" 200 1219
2021-12-09 18:32:10 - INFO - "GET /static/style.css HTTP/1.1" 200 648
2021-12-09 18:32:10 - INFO - "GET /favicon.ico HTTP/1.1" 200 1739
2021-12-09 18:45:32 - INFO - "GET / HTTP/1.1" 200 1219
2021-12-09 18:45:32 - INFO - "GET /static/style.css HTTP/1.1" 304 0
2021-12-09 18:45:32 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:02:21 - INFO - "GET / HTTP/1.1" 200 1219
2021-12-09 19:02:21 - INFO - "GET /static/style.css HTTP/1.1" 304 0
2021-12-09 19:02:21 - INFO - "GET / HTTP/1.1" 200 1219
2021-12-09 19:02:21 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:02:21 - WARNING - "GET /turtle HTTP/1.1" 404 1536
2021-12-09 19:02:21 - INFO - "GET /track HTTP/1.1" 200 860
2021-12-09 19:02:21 - INFO - "GET /track/1 HTTP/1.1" 200 430
2021-12-09 19:02:21 - INFO - "GET /track/2 HTTP/1.1" 200 426
2021-12-09 19:02:21 - INFO - "GET /track HTTP/1.1" 200 860
2021-12-09 19:12:24 - INFO - "GET /track HTTP/1.1" 200 860
2021-12-09 19:12:24 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:12:36 - INFO - "GET /track HTTP/1.1" 200 860
2021-12-09 19:12:36 - INFO - "GET /track/1 HTTP/1.1" 200 430
2021-12-09 19:13:33 - ERROR - UnboundLocalError in /Users/matthewlaude/development ðŸ‘¾/SEIR_penguin/PROJECTS/project4/backend/app/http/controllers/TrackController.py on line 24
2021-12-09 19:13:33 - ERROR - "POST /track HTTP/1.1" 500 1041593
2021-12-09 19:13:50 - ERROR - UnboundLocalError in /Users/matthewlaude/development ðŸ‘¾/SEIR_penguin/PROJECTS/project4/backend/app/http/controllers/TrackController.py on line 24
2021-12-09 19:13:50 - ERROR - "POST /track HTTP/1.1" 500 1041593
2021-12-09 19:15:18 - ERROR - NameError in /Users/matthewlaude/development ðŸ‘¾/SEIR_penguin/PROJECTS/project4/backend/app/http/controllers/TrackController.py on line 24
2021-12-09 19:15:18 - ERROR - "POST /track HTTP/1.1" 500 973578
2021-12-09 19:15:25 - ERROR - NameError in /Users/matthewlaude/development ðŸ‘¾/SEIR_penguin/PROJECTS/project4/backend/app/http/controllers/TrackController.py on line 24
2021-12-09 19:15:25 - ERROR - "POST /track HTTP/1.1" 500 973578
2021-12-09 19:15:26 - ERROR - NameError in /Users/matthewlaude/development ðŸ‘¾/SEIR_penguin/PROJECTS/project4/backend/app/http/controllers/TrackController.py on line 24
2021-12-09 19:15:26 - ERROR - "POST /track HTTP/1.1" 500 973578
2021-12-09 19:15:52 - ERROR - NameError in /Users/matthewlaude/development ðŸ‘¾/SEIR_penguin/PROJECTS/project4/backend/app/http/controllers/TrackController.py on line 24
2021-12-09 19:15:52 - ERROR - "POST /track HTTP/1.1" 500 973578
2021-12-09 19:16:08 - ERROR - QueryException in 0 on line 0
2021-12-09 19:16:08 - ERROR - "POST /track HTTP/1.1" 500 983106
2021-12-09 19:16:26 - ERROR - QueryException in 0 on line 0
2021-12-09 19:16:26 - ERROR - "POST /track HTTP/1.1" 500 983106
2021-12-09 19:17:15 - ERROR - QueryException in 0 on line 0
2021-12-09 19:17:15 - ERROR - "POST /track/ HTTP/1.1" 500 983110
2021-12-09 19:18:54 - ERROR - QueryException in 0 on line 0
2021-12-09 19:18:54 - ERROR - "POST /track/ HTTP/1.1" 500 982980
2021-12-09 19:24:08 - INFO - "POST /track/ HTTP/1.1" 200 216
2021-12-09 19:24:08 - INFO - "GET /track HTTP/1.1" 200 1078
2021-12-09 19:24:08 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:24:59 - ERROR - JSONDecodeError in /Users/matthewlaude/.pyenv/versions/3.9.6/lib/python3.9/json/decoder.py on line 353
2021-12-09 19:24:59 - ERROR - "POST /track/ HTTP/1.1" 500 1043673
2021-12-09 19:25:38 - ERROR - JSONDecodeError in /Users/matthewlaude/.pyenv/versions/3.9.6/lib/python3.9/json/decoder.py on line 353
2021-12-09 19:25:38 - ERROR - "POST /track/ HTTP/1.1" 500 1043673
2021-12-09 19:27:29 - INFO - "GET /track HTTP/1.1" 200 1296
2021-12-09 19:27:29 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:27:29 - WARNING - "GET /track//3 HTTP/1.1" 404 1536
2021-12-09 19:27:29 - INFO - "GET /track/3 HTTP/1.1" 200 216
2021-12-09 19:27:29 - INFO - "GET /track HTTP/1.1" 200 1296
2021-12-09 19:27:29 - INFO - "GET /track HTTP/1.1" 200 1296
2021-12-09 19:27:29 - WARNING - "PUT /track/3 HTTP/1.1" 404 1536
2021-12-09 19:28:31 - INFO - "PUT /track/3 HTTP/1.1" 200 244
2021-12-09 19:28:31 - INFO - "GET /track HTTP/1.1" 200 1322
2021-12-09 19:28:31 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:28:31 - INFO - "PUT /track/4 HTTP/1.1" 200 249
2021-12-09 19:28:31 - INFO - "GET /track HTTP/1.1" 200 1353
2021-12-09 19:31:18 - INFO - "POST /track HTTP/1.1" 200 247
2021-12-09 19:31:18 - INFO - "GET /track HTTP/1.1" 200 1602
2021-12-09 19:31:18 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:31:53 - ERROR - AttributeError in 0 on line 0
2021-12-09 19:31:53 - ERROR - "DELETE /track/5 HTTP/1.1" 500 1041693
2021-12-09 19:32:01 - ERROR - AttributeError in 0 on line 0
2021-12-09 19:32:01 - ERROR - "DELETE /track/5 HTTP/1.1" 500 1041693
2021-12-09 19:32:39 - INFO - "DELETE /track/5 HTTP/1.1" 200 249
2021-12-09 19:32:39 - INFO - "GET /track HTTP/1.1" 200 1353
2021-12-09 19:32:39 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-09 19:32:39 - INFO - "DELETE /track/3 HTTP/1.1" 200 244
2021-12-09 19:32:39 - INFO - "GET /track HTTP/1.1" 200 1109
2021-12-09 19:33:36 - ERROR - ResponseError in 0 on line 0
2021-12-09 19:33:40 - ERROR - ResponseError in 0 on line 0
2021-12-11 15:47:17 - INFO - "GET / HTTP/1.1" 200 1219
2021-12-11 15:47:17 - INFO - "GET /static/style.css HTTP/1.1" 304 0
2021-12-11 15:47:17 - WARNING - "GET /turtle HTTP/1.1" 404 1536
2021-12-11 15:47:17 - INFO - "GET /track HTTP/1.1" 200 1120
2021-12-11 15:47:17 - INFO - "GET /track HTTP/1.1" 200 1120
2021-12-11 15:47:17 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-11 15:47:17 - INFO - "GET /track/1 HTTP/1.1" 200 565
2021-12-11 15:47:17 - INFO - "GET /track/2 HTTP/1.1" 200 551
2021-12-11 15:47:17 - INFO - "GET /track/2 HTTP/1.1" 200 551
2021-12-11 15:47:17 - INFO - "GET /track/2 HTTP/1.1" 200 551
2021-12-11 15:47:17 - INFO - "GET /favicon.ico HTTP/1.1" 304 0
2021-12-11 15:47:17 - INFO - "GET /track HTTP/1.1" 200 1120
2021-12-11 15:56:07 - ERROR - JSONDecodeError in /Users/matthewlaude/.pyenv/versions/3.9.6/lib/python3.9/json/decoder.py on line 353
2021-12-11 15:56:07 - ERROR - "POST /track HTTP/1.1" 500 1044460
